duckduckgo/Android
# Update to autofill 3.0.0

Task/Issue URL: https://app.asana.com/0/1198964220583541/1200553245468873
Tech Design URL: 
CC: 

**Description**:
Updates autofill to 3.0.0

**Steps to test this PR**:
1. Checkout branch
1. Run git submodule
1. Confirm autofill commit hash is`3f8b4d63731d0f4417dca569de69acc39a67f641`

**Autofill still works**
1. See https://app.asana.com/0/1198964220583541/1200553245468873

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
